data/mods/Perk_melee/PERKS/perks.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "type": "mutation_type",
+    "id": "perk"
+  },
```

Comment:
Is this needed?

Reply:
I think its required for purifier reasons IRC its something bombasticperks found out.